MennaMohie/assignment1/Q1.cpp

```diff
@@ -23,7 +22,15 @@ int main()
         cout<<"Please type 's' for Case Sensitive Anagram Checking or 'i' for Case Insensitive Anagram Checking: ";
         cin>>sensitivity;
 
-        cout<<(anagramWordCheck(word1, word2, tolower(sensitivity))?"The two words are anagrams.":"The two words are not anagrams.");
+        bool caseSensitive;
+        sensitivity=tolower(sensitivity);
+        if(sensitivity=='i')
+            caseSensitive=false;
+        else if (sensitivity=='s')
+            caseSensitive=true;
```

Comment:
else?
I think lines 25-30 can be more clear as:
`bool caseSensitive = tolower(sensitivity) == 's'`